### ISSUE TITLE
Game sometimes won't load any searches

### ISSUE THREAD

**Issue by ReNForce72** [111116/osu#70]
**Describe the bug**
Every once in a while, I open the site and search for a song, but when I hit enter, I get the "加载更多" message box, but when I click on it, nothing happens; like the game is stuck. I also can't load my favorites or any of the songs listed in the "what happened" playlist.

This happens to about 5% of all my play sessions, and from what I remember, it lasts anywhere from 1 to 3 days, and then the issue resolves itself. I've tried to clear my cache and browser data, but that doesn't seem to do anything. Neither does turning off my Ad Blocker.

**Console error**
addbeatmaplist.js:350 GET https://api.sayobot.cn/beatmaplist?0=20&1=20&2=4&3=Tsuyogari%20(TV%20Size)&5=1 net::ERR_CONNECTION_TIMED_OUT

**Desktop (please complete the following information):**
 - OS: macOS (Mojave 10.14.6)
 - Browser: Chrome (Version 90.0.4430.93  (Official Build) (x86_64))

Also, side note; completely unrelated: how'd you guys come up with Osaka Salmon University haha, just curious.

**Comment by 111116** [comment]
Thanks for your detailed report.
During periods when searching doesn't work, will other playlists such as "popular" and "latest" load?

**Comment by 111116** [comment]
As for the site name, we're just making up some random name that abbreviates to "osu" (and I personally love Japanese food), in tribute to the real OSU! but trying to avoid trademark infringement.

**Comment by ReNForce72** [comment]
Nope, those playlists are blank as well.
![Screen Shot 2021-05-05 at 10 33 35 PM](https://user-images.githubusercontent.com/83483901/117238224-1cb13b00-adf2-11eb-985f-ee228b0ad32b.png)
that's more or less all I get on any tab, aside from settings and FAQ tabs. Those load everything up just fine.

**Comment by 111116** [comment]
Likely there's a network issue between your PC and `api.sayobot.cn`, the beatmap backend. May I ask what region do you live in, and are you using any proxies? I'll feedback to manager of `sayobot.cn` to see if there's a solution.

**Comment by ReNForce72** [comment]
I'm in the U.S., and I'm not sure if I'm using a proxy server or not, but I would assume that I'm not; I don't have a VPN or anything set up. 

Edit: Unless 'Passive FTP mode' counts as a proxy, which I currently have enabled.

**Comment by BlaNKtext** [comment]
> Likely there's a network issue between your PC and `api.sayobot.cn`, the beatmap backend. May I ask what region do you live in, and are you using any proxies? I'll feedback to manager of `sayobot.cn` to see if there's a solution.

It is definitely a problem with `sayobot.cn`, I host a translated version of the site and lately beatmaps dont load at all in the home page or anywhere else, the issue is usually fixed in a few hours (today there was a outage for around 3 hours)

**Comment by ReNForce72** [comment]
looks like the problem is gone now. I'll reopen the issue if it comes back with updated specs if needed

**Comment by ReNForce72** [comment]
Well that was quick. Not sure why it's happening again. Same hardware, same software versions.

The issue seemed to have started happening more frequently ever since I downloaded Osu Lazer. I have no idea why this would cause any issues, and I don't think it is, but that's really the only change I can think of that could somehow be related.

**Comment by BlaNKtext** [comment]
> Well that was quick. Not sure why it's happening again. Same hardware, same software versions.
> 
> The issue seemed to have started happening more frequently ever since I downloaded Osu Lazer. I have no idea why this would cause any issues, and I don't think it is, but that's really the only change I can think of that could somehow be related.

Its not a problem on your end, its a problems with the sayobot beatmap API, 111116 can't do anything about it either.

**Comment by 111116** [comment]
yep. Sayobot guys are informed and are now working on it. Feel free to keep track of this issue here.

**Comment by ReNForce72** [comment]
round 3 then, I suppose, haha. 
![Screen Shot 2021-05-17 at 9 53 03 AM](https://user-images.githubusercontent.com/83483901/118509533-ba134580-b6f5-11eb-9b8c-14e3c4bb71d2.png)

**Comment by soviet-chungus** [comment]
Welp the same thing is happening to me now
